src/platform/unittest/TlsTest.cpp

```diff
@@ -1777,6 +1777,26 @@ TEST_F(TlsTest, ClientCertificateDeferValidation)
     DoHandshake(ServerContext, ClientContext);
 }
 
+TEST_F(TlsTest, ClientCertificateDeferValidationNoCert)
+{
+    CxPlatClientSecConfig ClientConfig(
+        QUIC_CREDENTIAL_FLAG_NO_CERTIFICATE_VALIDATION
+#ifdef QUIC_ENABLE_ANON_CLIENT_AUTH_TESTS
+        | QUIC_CREDENTIAL_FLAG_USE_SUPPLIED_CREDENTIALS
+#endif
```

Comment:
Why modify the behavior of a test with the `QUIC_ENABLE_ANON_CLIENT_AUTH_TESTS` and not have two separate tests?

Reply:
USE_SUPPLIED_CREDS is necessary for schannel, and is an schannel-only flag. I want this test to also cover OpenSSL.